app/lib/forms/choose_your_npq.rb

```diff
@@ -74,6 +85,14 @@ def course
 
   private
 
+    def lead_provider_valid?
```

Comment:
could this not have been added as an active model validation on the form?

Reply:
It's not really a validation, it's more checking a status to direct behaviour of the form.

I suppose we could throw an error message but then we're telling the user to go change their provider and come back to change their NPQ which is clunky and in certain circumstances may not make sense or be easily manageable. Going down this path of having it just clear out the data and allowing the user to progress throws less roadblocks in front of progression.